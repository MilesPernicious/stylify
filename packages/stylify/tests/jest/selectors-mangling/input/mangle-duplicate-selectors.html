<div class=" justify-content:space-between "></div>
<div class="justify-content:space-between">
<div class='justify-content:space-between justify-content:space-between '>
