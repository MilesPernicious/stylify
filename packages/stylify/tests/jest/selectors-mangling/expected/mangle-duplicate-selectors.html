<div class=" a "></div>
<div class="a">
<div class='a a '>
